script.js

```diff
@@ -1 +1,89 @@
-console.log('dev')
\ No newline at end of file
+
+'use strict'
+
+// DOM ELLEMENTS
+const body = document.querySelector('body')
+const home = document.querySelector('.home')
+const theme = document.querySelector('.theme-api');
+const searchInput = document.querySelector('.src-input');
+const searchBtn = document.querySelector('.src-btn');
+const content = document.querySelector('.content-container')
+
+const user = document.querySelector('.username');
+const url = document.querySelector('.link');
+const joinDate = document.querySelector('.join-date');
+const info = document.querySelector('.info');
+const avatar = document.querySelector('.avatar')
+
+const reposNum = document.querySelector('.repos');
+const followersNum = document.querySelector('.followers');
+const followingNum = document.querySelector('.following');
+
+const city = document.querySelector('.location');
+const link = document.querySelector('.website');
+const twitter = document.querySelector('.twitter');
+const company = document.querySelector('.company');
+
+// FUNCTIONS
+function makeRequest() {
+    getData(searchInput.value)
+    searchInput.value = null;
+}
+
+function getData(username) {
+    fetch(`https://api.github.com/users/${username}`)
+    .then(res => res.json())
+    .then(data => {
+        const userData = data;
+        console.log(userData)
+        renderData(userData);
+        // content.classList.remove('hidden')
+    })
+}
+
```

Comment:
Perform validation, if @ is entered. Write to the user how to enter a valid username when it makes a mistake.
The rest is great!

Reply:
![image](https://user-images.githubusercontent.com/90243506/135731451-fa262da8-543a-4794-b7f8-a79359a578bd.png)
Add more by clicking on Github results opens a link to the profile in the new tab.